source/slang/slang-type-layout.h

```diff
@@ -392,7 +384,7 @@ class TypeLayout : public Layout
 typedef unsigned int VarLayoutFlags;
 enum VarLayoutFlag : VarLayoutFlags
 {
-    HasSemantic = 1 << 1
+    HasSemantic = 1 << 0
```

Comment:
Note: I think the flag had that value simply because there used to be another flag with `1 << 0` as the value. Since this isn't part of the binary interface to Slang, though, there is no reason not to clean it up like this.